package.json

```diff
@@ -30,7 +30,7 @@
     "stylus": "^0.50.0",
     "superagent": "~0.15.7",
     "through2": "^0.6.3",
-    "workshopper": "^2.3.1",
-    "workshopper-exercise": "^2.3.0"
+    "workshopper": "^2.6.0",
+    "workshopper-exercise": "^2.4.0"
```

Comment:
Its not about translation. but, It needed for menus.
ref workshopper/organizers#2